build.d/80-dependencies

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# Copyright 2017 LasLabs Inc.
+
+import os
+import subprocess
+
+
+DIR_CUSTOM = '/opt/odoo/custom/dependencies'
+
+REQ_TYPES = {
+    'apt': 'apt-get install -y',
+    'pip': 'pip install',
+    'npm': 'npm install -g',
+    'gem': 'gem install',
+}
+
+
+def do_install(command, requirements_file):
+    with open(requirements_file, 'r') as fh:
+        requirements = fh.read().strip()
+    command += ' %s' % requirements
+    proc = subprocess.Popen(command.split(' '))
```

Comment:
if you call split without arguments, in case there are multiple spaces it will split correctly, and strip before:

```python
>>> "   asdfasdf  a sfdasdf    ".split(" ")
['', '', '', 'asdfasdf', '', 'a', 'sfdasdf', '', '', '', '']
>>> "   asdfasdf  a sfdasdf    ".split()
['asdfasdf', 'a', 'sfdasdf']
```